charmcraft.yaml

```diff
@@ -17,7 +17,7 @@ parts:
 bases:
   - build-on:
       - name: ubuntu
-        channel: "20.04"
+        channel: "22.04"
```

Comment:
according to the run-on section, the intention is to support focal and jammy, so the build-on should happen on focal (20.04), because we have better chances of getting a working charm when built on python-3.8 (focal) and run it on python-3.10 (jammy), than the other way around.

Reply:
Changed applied, it is now being built from 20.04.
I could confirm the charm still work correctly on Focal and Jammy.